evm-adapters/testdata/CheatCodes.sol

```diff
@@ -186,6 +186,25 @@ contract CheatCodes is DSTest {
         prank.bar(address(this));
     }
 
+    function testPayable() public {
```

Comment:
can we change to `testPrankPayable` just so you can run  `-m testPrank` and test all prank related cheatcodes?

Reply:
done